kubernetes/templates/metrics-aggregator.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: oisp.org/v1
+kind: BeamService
+metadata:
+  name: metrics-aggregator
+spec:
+  entryClass: "org.oisp.services.MetricsAggregator"
+  args:
+    runner: FlinkRunner
+    streaming: "true"
+    metricsTopic: {{ .Values.metricsTopic | quote }}
+    bootstrapServers: {{ .Values.kafka.service}}
+  package:
+    url: {{ .Values.metricsAggregator.url }}
+    username: {{ .Values.metricsAggregator.ftpUsername }}
```

Comment:
username and password are not required because we are not using ftp.

Reply:
ok -removed